src/stravalib/client.py

```diff
@@ -588,6 +588,38 @@ def get_club_activities(
             limit=limit,
         )
 
+    def get_club_admins(
+        self, club_id: int, limit: int | None = None
+    ) -> BatchedResultsIterator[model.SummaryAthlete]:
+        """Gets the activities associated with specified club.
```

Comment:
```suggestion
        """Returns a list of the administrators of a given club.
```

i think?  i copied this from swagger.json